src/component/AnimeList.jsx

```diff
@@ -0,0 +1,27 @@
+const AnimeList = ({ apiData }) => {
+  if (apiData.status === 404) {
+    return <h1>{apiData.message}</h1>;
+  } else {
```

Comment:
Should not handle API error here in the component. It should be handled where it get called

Reply:
Implemented.